Makefile

```diff
@@ -86,7 +87,7 @@ docker-push:
 controller-gen:
 ifeq (, $(shell which controller-gen))
 	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.0-beta.2
-CONTROLLER_GEN=$(shell go env GOPATH)/bin/controller-gen
+CONTROLLER_GEN=$(shell which controller-gen)
```

Comment:
This assumes that `PATH` includes `$GOPATH/bin` which isn't always the case. Why was `$(shell go env GOPATH)/bin/` not a suitable option? `go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.0-beta.2` would install this in the appropriate `go env GOPATH`, right?

Reply:
For `'test-integration` step, the value of `go env GOPATH` is unfortunately this: `/home/circleci/.go_workspace:/usr/local/go_workspace` - there are two folder paths separated with a colon. So the concatenation of `/bin/controller-gen` wasn't working.

Reply:
Oh I see - ok that makes sense now!